appserver/ha/ha-file-store/src/main/java/org/glassfish/ha/store/adapter/file/FileBackingStore.java

```diff
@@ -427,17 +388,6 @@ private void writetoFile(K sessionKey, String fileName, byte[] data)
             }
             String errMsg = "Could not save session: " + sessionKey;
             throw new BackingStoreException(errMsg, ex);
-        } finally {
```

Comment:
Should we keep the failure logging here with a catch?

Reply:
I wouldn't say so. Its logging of exceptions occurring in something that is backed into try-with-resource. I would think that these were only added because the author didn't want to just ignore a cached exception.

Reply:
Maybe a further clarification: Using try-with-resources is a change of behaviour since exceptions occurring while closing will no longer be caught. My assumptions here was that the author was using the replaced code as the pattern to use for IO and I replaced it with the new pattern, the try-with-resouce.